src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PSBwaAlignerSpark.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.spark.pathseq;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.hellbender.utils.bwa.BwaMemIndexSingleton;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+/**
+ * Wrapper class for using the PathSeq Bwa aligner class in Spark. Encapsulates closing the index when done.
+ */
+public final class PSBwaAlignerSpark implements AutoCloseable {
+
+    private final JavaSparkContext ctx;
+    private final Broadcast<PSBwaArgumentCollection> bwaArgsBroadcast;
+
+    public PSBwaAlignerSpark(final JavaSparkContext ctx, final PSBwaArgumentCollection bwaArgs) {
+        this.ctx = ctx;
+        this.bwaArgsBroadcast = ctx.broadcast(bwaArgs);
+    }
+
+    private static JavaRDD<GATKRead> doBwaAlignmentHelper(final JavaRDD<GATKRead> reads,
```

Comment:
Don't see how this 1-liner is helpful.  Was it for testing?  Why not just put this code directly into doBwaAlignment.